appintro/src/main/java/com/github/appintro/AppIntro.kt

```diff
@@ -199,4 +200,22 @@ abstract class AppIntro : AppIntroBase() {
             bottomSeparator.visibility = View.INVISIBLE
         }
     }
+
+    /**
+     * Adds a margin to the bottom of the content of the slide equal to the bottom bar.
+     * For situations where the bar is not transparent, and you want the content of the slide to
+     * be intractable.
+     *
+     * @param setBarPadding Set : true for margin. false for no margin.
```

Comment:
Here you mix `margin` and `padding`. Let's stick to one (margin is the correct one here).